plugin-template

```diff
@@ -105,21 +101,25 @@ DEPRECATED_FILES = {
         ".ci/scripts/update_github.sh",
         ".ci/scripts/update_redmine.sh",
         ".ci/scripts/upper_bound.py",
+        ".github/workflows/changelog.yml",
         ".github/workflows/fips.yml",
         ".github/workflows/scripts/create_release_from_tag.sh",
         ".github/workflows/scripts/install_python_client.sh",
         ".github/workflows/scripts/install_ruby_client.sh",
         ".github/workflows/scripts/release.py",
         ".github/workflows/scripts/stage-changelog-for-master.py",
+        ".github/workflows/scripts/post_docs_test.sh",
         ".github/workflows/scripts/update_ci.sh",
+        ".github/workflows/scripts/docs-publisher.py",
+        ".github/workflows/scripts/publish_docs.sh",
         ".travis",
         ".travis.yml",
         "flake8.cfg",
         "CHANGES/.TEMPLATE.rst",
     ],
     "docs": [
-        "docs/_static/.this_makes_it_so_git_keeps_the__static_folder",
-        "docs/_static/api.json",
+        "docs/_static/",
+        "docs/_scripts/",
```

Comment:
Eventually we'll start renaming `staging_docs` to `docs` on plugins, thus removing all legacy docs. The dirs `_static` and `_scripts` have no special meaning anymore, but I can see someone keeping images on a `_static`, for example. It could be a surprise if CI tried to remove it.

I think we should not include any thing in the docs deprecation list.

Reply:
I reverted this change. Thanks for noticing!